src/Sulu/Bundle/AdminBundle/Resources/js/components/Snackbar/Snackbar.js

```diff
@@ -7,9 +7,11 @@ import {translate} from '../../utils/Translator';
 import Icon from '../Icon';
 import snackbarStyles from './snackbar.scss';
 
-export type SnackbarType = 'error' | 'warning';
+export type SnackbarType = 'error' | 'warning' | 'info' | 'success';
 
 type Props = {|
+    behaviour: 'static' | 'floating',
```

Comment:
this prop does not really change any behaviour, right? 🙈

Reply:
it affects the style and the text, right? i think we have `skin` on a few other components. think that would describe it better

Reply:
Is fine for me, was also unsure about it.